backend/src/property_handlers.py

```diff
@@ -14,7 +16,55 @@ def get_property():
     :param request: dictionary of jwtPayload
     :return tuple (response body (dict), response code (int), error (string))
     """
-    return jsonify(code=500, error='not implemented'), 500
+    try:
+        property_list = app.services.property_service().get_all_property()
+        if property_list:
+            return jsonify(property_list), 200
+        else:
+            return jsonify(code=400, error='error processing request'), 400
+    except:
+        jsonify(code=500, error='internal error'), 500
+    return jsonify(code=500, error='internal error'), 500
+
+
+def extract_property_info(prop_data):
```

Comment:
Hey, I was thinking about a better way to handle this sort of thing after we discussed it last hack night. I have a draft PR (https://github.com/code-for-charlottesville/housinghub/pull/37) for using a declarative approach to request/response serialization.